### DISCUSSION TITLE
GBA Mus Riper Inside VGMTrans

### DISCUSSION THREAD

**Discussion by RetroGamesClub** [vgmtrans/vgmtrans#520] [General]
Could it be possible to integrate Latest GBA_Mus_Reaper release (I think 3.3) inside VGM Trans QT, to extrace Midi and SF2 file from GBA games which work with sappy engine lite DS games?  Thank you.

**Comment by sykhro** [comment]
We already have it!

**Reply by RetroGamesClub** [reply]
Mmm... I've extracted a game manually, with Gba mus reaper but if i load the same GBA file inside VGMTrans, I've nothing created. So I'm sure it's appy engine, but i can not see anything. 
(Another question, what about gsflib?)

**Reply by sykhro** [reply]
We have that too, file an issue

**Reply by RetroGamesClub** [reply]
hi, yesterday I've provided info.
